src/services/healthcareProfessionalService.ts

```diff
@@ -681,8 +681,18 @@ function validateUpdateProfessionalInput(input: Partial<gqlTypes.UpdateHealthcar
         errors: []
     }
 
+    if (Object.keys(input).length < 1) {
+        validationResults.hasErrors = true
+        validationResults.errors?.push({
+            field: 'input',
+            errorCode: ErrorCode.MISSING_INPUT,
+            httpStatus: 400
+        })
+        return validationResults
+    }
+
     //business rule: at least one facility id is required
-    if (!input.facilityIds || input.facilityIds.length < 1) {
+    if (input.facilityIds && input.facilityIds.length < 1) {
```

Comment:
I don't think this logic really makes sense anymore actually since the facilityIds here are facilityId updates. 
I think the logic here would be to check if the updates are making the facilityId count less than 1.